codegen-server/python/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+
+plugins {
+    kotlin("jvm")
+    maven
+    `maven-publish`
+}
+
+description = "Generates Rust/Python server-side code from Smithy models"
+
+extra["displayName"] = "Smithy :: Rust :: Codegen :: Server :: Python"
+
+extra["moduleName"] = "software.amazon.smithy.rust.codegen.server.python"
+
+group = "software.amazon.smithy.rust.codegen.server.python.smithy"
+
+version = "0.1.0"
+
+val smithyVersion: String by project
+val kotestVersion: String by project
+
+dependencies {
+    implementation(project(":codegen"))
+    implementation(project(":codegen-server"))
+    implementation("software.amazon.smithy:smithy-aws-traits:$smithyVersion")
+    implementation("software.amazon.smithy:smithy-protocol-test-traits:$smithyVersion")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.6.1")
+    testImplementation("io.kotest:kotest-assertions-core-jvm:$kotestVersion")
+}
+
+// unlike the client-runtime, software.amazon.smithy.rust.codegen.smithy-kotlin codegen package is
+// not expected to run on Android...we can target 1.8
+tasks.compileKotlin { kotlinOptions.jvmTarget = "1.8" }
+
+tasks.compileTestKotlin { kotlinOptions.jvmTarget = "1.8" }
+
+// Reusable license copySpec
+val licenseSpec = copySpec {
+    from("${project.rootDir}/LICENSE")
+    from("${project.rootDir}/NOTICE")
+}
+
+// Configure jars to include license related info
+tasks.jar {
+    metaInf.with(licenseSpec)
+    inputs.property("moduleName", project.name)
+    manifest { attributes["Automatic-Module-Name"] = project.name }
+}
+
+val sourcesJar by tasks.creating(Jar::class) {
+    group = "publishing"
+    description = "Assembles Kotlin sources jar"
+    classifier = "sources"
+    from(sourceSets.getByName("main").allSource)
+}
+
+tasks.test {
+    useJUnitPlatform()
+    testLogging {
+        events("passed", "skipped", "failed")
+        exceptionFormat = TestExceptionFormat.FULL
+        showCauses = true
+        showExceptions = true
+        showStackTraces = true
+        showStandardStreams = true
+    }
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("default") {
+            from(components["java"])
+            artifact(sourcesJar)
+        }
+    }
+    repositories { maven { url = uri("$buildDir/repository") } }
+}
```

Comment:
Should this get set up later?

Reply:
This looks like it's config to publish jars to Maven, but we don't actually publish to Maven yet. I don't think it harms anything though 🤔

Reply:
The same is also present inside `codegen` and `codegen-server` `build.gradle.kt`. Honestly I have just copied it from the `codegen` folder.